x-pack/plugins/lens/public/drag_drop/drag_drop.scss

```
@@ -52,3 +47,32 @@
     text-decoration: line-through;
   }
 }
+
+.lnsDragDrop-reorderableContainer {
```

Just nit-picking on BEM naming here
```suggestion
.lnsDragDrop__reorderableContainer {
```